### ISSUE TITLE
Importing these projects into Eclipse

### ISSUE THREAD

**Issue by benwarrick** [PacktPublishing/Hands-On-Microservices-with-Spring-Boot-and-Spring-Cloud#22]
Is there a way to import these project into Eclipse? I'm trying to work with chapter 3, 2-basic-rest-service. I use Import and then Gradle project, but it ends up importing a "2-basic-service" project filled with broker references, and then each package is imported as a separate project.

**Comment by magnus-larsson** [comment]
Hello Benjamin!

Sorry for not responding earlier, but I have been very busy lately finishing [the 2'nd edition of my book](https://www.amazon.com/Microservices-Spring-Boot-Cloud-microservices/dp/1801072973).  
(I have also written a [blog post on the updates in the 2nd edition](https://callistaenterprise.se/blogg/teknik/2021/08/09/microservices-book-2nd-edition-updates/), if you are interested)

I'm not using Eclipse myself, so I downloaded Eclipse version 2021-06 (4.20.0) and tried to import `chapter 3, 2-basic-rest-service`. It failed with the error message: 

`org.gradle.tooling.BuildException: Could not fetch model of type 'GradleBuild' using connection to Gradle distribution 'https://services.gradle.org/distributions/gradle-5.6.4-bin.zip'.`

Not sure if it is similar to the problem you experienced?

However, I googled a bit and found a lot of issues discussed around this error message and they seems to relate to version missmatch between Gradle and Eclipse in many cases...

So I upgraded Gradle to the latest version 7.1.1 and imported the project again, and this time it seemed to work:
(Changed 5.6.4 to 7.1.1 in the file `Chapter03/2-basic-rest-services/gradle/wrapper/gradle-wrapper.properties`)

<img width="1099" alt="Screenshot 2021-08-12 at 13 37 11" src="https://user-images.githubusercontent.com/1313727/129190901-2e433622-2301-4ca2-9f74-962814220b7d.png">

Hope this helps!
